target/linux/ramips/image/mt7621.mk

```diff
@@ -2109,6 +2109,16 @@ define Device/rostelecom_rt-sf-1
 endef
 TARGET_DEVICES += rostelecom_rt-sf-1
 
+define Device/ruijie_rg-ew1200g-pro
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 16384k
```

Comment:
I didn't explain it enough and I mislead you, but it should be the same size as A.

Reply:
I may be dump, but what's A?

Reply:
Sorry, it's my bad when copied&pasted from Google Translator...